scripts/generate_manifest.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+import itertools
+from openpyxl import load_workbook
+import os
+
+from EPPs.common import SendMailEPP
+
+
+class GenerateManifest(SendMailEPP):
```

Comment:
This does not need to subclass SendMailEPP

Reply:
updated